astronomer/astronomer.tf

```
@@ -0,0 +1,133 @@
+# Initialize kubectl
+
+provider "kubernetes" {
+  config_path      = "./kubeconfig"
+  load_config_file = true
+}
+
+# Create prerequisite resources
+
+resource "kubernetes_service_account" "tiller" {
+  metadata {
+    name      = "tiller"
+    namespace = "kube-system"
+  }
+}
+
+resource "kubernetes_cluster_role_binding" "tiller-binding" {
+  depends_on = ["kubernetes_service_account.tiller"]
+
+  metadata {
+    name = "tiller-binding"
+  }
+
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "cluster-admin"
+  }
+
+  subject {
+    kind      = "ServiceAccount"
+    name      = "tiller"
```

Can be updated to:

```suggestion
    name      = "${kubernetes_service_account.tiller.name}"
```

If you do that you can remove the explicit dependency: `depends_on = ["kubernetes_service_account.tiller"]`

---

this didn't work for me